docs/quickstart/installation.md

```diff
@@ -157,50 +161,76 @@ docker run --gpus all \                                         # Give the conta
-- To avoid memory issues or limitations during processing, it is recommended to use either ```--shm-size=12gb``` or ```--ipc=host``` to increase the memory available to the docker container. 12gb as in the example is only a suggestion and may be replaced by other values depending on your hardware and requirements. 
+- The current version of docker is built for multi-architecture (CUDA architectures) use. The target architecture(s) must be defined at build time for Colmap and tinyCUDNN to be able to compile properly. If your GPU architecture is not covered by the following table you need to replace the number in the line `ARG CUDA_ARCHITECTURES=90;89;86;80;75;70;61;52;37` to your specific architecture. It also is a good idea to remove all architectures but yours (e.g. `ARG CUDA_ARCHITECTURES=86`) to speedup the docker build process a lot.
+- To avoid memory issues or limitations during processing, it is recommended to use either `--shm-size=12gb` or `--ipc=host` to increase the memory available to the docker container. 12gb as in the example is only a suggestion and may be replaced by other values depending on your hardware and requirements.
 
 **Currently supported CUDA architectures in the docker image**
 
-GPU | CUDA arch
--- | --
-H100 | 90
-40X0 | 89
-30X0 | 86
-A100 | 80
-20X0 | 75 
-TITAN V / V100 | 70
-10X0 / TITAN Xp | 61
-9X0 | 52
-K80 | 37
+(tiny-cuda-arch-list)=
+
+| GPU             | CUDA arch |
+| --------------- | --------- |
+| H100            | 90        |
+| 40X0            | 89        |
+| 30X0            | 86        |
+| A100            | 80        |
+| 20X0            | 75        |
+| TITAN V / V100  | 70        |
+| 10X0 / TITAN Xp | 61        |
+| 9X0             | 52        |
+| K80             | 37        |
 
 ## Installation FAQ
 
+- [ImportError: DLL load failed while importing \_89_C](tiny-cuda-mismatch-arch)
 - [TinyCUDA installation errors out with cuda mismatch](tiny-cuda-mismatch-error)
 - [TinyCUDA installation errors out with no CUDA toolset found](tiny-cuda-integration-error)
 - [Installation errors, File "setup.py" not found](pip-install-error)
 - [Runtime errors, "len(sources) > 0".](cuda-sources-error)
 
  <br />
 
+(tiny-cuda-mismatch-arch)=
+
+**ImportError: DLL load failed while importing \_89_C**
+
+tiny-cuda-nn was installed for the wrong hardware architecture.
+
+**Solution**:
+
+Reinstall tiny-cuda with the following command:
+
+```bash
+TCNN_CUDA_ARCHITECTURES=XX pip install git+https://github.com/NVlabs/tiny-cuda-nn/#subdirectory=bindings/torch
```

Comment:
This doesn't work in Windows Command Prompt, for which the command is `set TCNN_CUDA_ARCHITECTURES=XX`